containers/DvmState.ts

```diff
@@ -39,25 +36,29 @@ const useContractState = () => {
   // get state from EMP
   const queryState = async () => {
     if (
-      dvm !== null &&
+      votingContract !== null &&
+      storeContract !== null &&
+      collAddress !== null &&
       priceIdentifier !== null &&
       expirationTimestamp !== null
     ) {
       const res = await Promise.all([
-        dvm.hasPrice(
+        votingContract.hasPrice(
```

Comment:
rather than calling it `dvm` I thought it cleaner to refer to the `votingContract` and `storeContract` as this now stores two contracts.